amq-message-age/README.md.jinja

```diff
@@ -50,11 +59,12 @@ These dashboards accompany [SignalFx's ActiveMQ integration](https://github.com/
 
 #### Configuring your endpoint
 
-Before we can send metrics to SignalFx, we need to make sure you are sending them to the correct SignalFx realm.
-To determine what realm you are in (YOUR_SIGNALFX_REALM), check your profile page in the SignalFx web application (click the avatar in the upper right and click My Profile).
-If you are not in the `us0` realm, you will need to set the `sfx_host` configuration option, as shown below.
-
-You will also need to set the `token` configuration option to your SignalFx organization access token (YOUR_SIGNALFX_API_TOKEN).
+Before you can send metrics to SignalFx, you need to make sure you are sending them to the correct SignalFx realm.
+To determine what realm you are in (YOUR_SIGNALFX_REALM), check your profile page in the SignalFx web application, as follows:
+1. Click the avatar in the upper right.
+2. Click My Profile.
```

Comment:
Use `menuselection` or bold for "My Profile." I think it's bold since this is markdown.

Reply:
done